backend/src/v2/compiler/tektoncompiler/testdata/importer.yaml

```diff
@@ -83,10 +83,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.uid
-          envFrom:
-          - configMapRef:
-              name: metadata-grpc-configmap
```

Comment:
Was this a new standard from kfp v2 or it's just used in this test case? I remember v1 was using env variable for looking up the metadata host.

Reply:
this part is not new. that configmap has been there for 3 yrs.

Reply:
I see, this was just under the integration test, but not the official deployment.